### DISCUSSION TITLE
Google Play

### DISCUSSION THREAD

**Discussion by frinpol** [mozilla-mobile/fenix#26233] [Q&A] [answered]
Why normal version of FF is 103.1.0 and the beta 103.0.0-*?  Shouldn't the beta version get farther release?

**Comment by kbrosnan** [comment, marked as answer]
There is no way to ship the exact same APK to two different play store listings. The Android app IDs are not the same so it would require two builds. A beta build of 103.1.0 would not be useful, within 2 days of the 103 release we release first 104 beta.
